pom.xml

```diff
@@ -148,7 +148,7 @@
     the link to metrics.dropwizard.io in docs/monitoring.md.
     -->
     <codahale.metrics.version>4.1.1</codahale.metrics.version>
-    <avro.version>1.10.1</avro.version>
+    <avro.version>1.10.2</avro.version>
```

Comment:
File a new ticket for Avro 1.10.2 upgrade?

Reply:
I was thinking about reusing the existing one to avoid the duplicated message in the release notes:
[SPARK-27733] Upgrade to Avro 1.10.1
[SPARK-2XXXX] Upgrade to Avro 1.10.2

WDTY @wangyum? Should I create it still?

Reply:
Ok rebasing to update the ticket number in the commit title.